packages/node-cli/src/commands/listobjects/ListObjectsInputHandler.js

```diff
@@ -147,9 +146,7 @@ module.exports = class ListObjectsInputHandler extends BaseInputHandler {
 			validate: (fieldValue) => showValidationResults(fieldValue, validateFieldIsNotEmpty),
 		};
 		questions.push(questionScriptId);
-
 		let answers = await prompt(questions);
-		answers[COMMAND_QUESTIONS_NAMES.AUTH_ID] = getProjectDefaultAuthId(this._executionPath);
```

Comment:
Maybe AUTH_ID could be removed from COMMAND_QUESTIONS_NAMES at the beginning of this file.